workers/transactor/src/transactor.ts

```diff
@@ -108,17 +102,23 @@ export class Transactor extends DurableObject<Env> {
     registerAdapterFactory('postgresql', createPostgresAdapter, true)
     registerDestroyFactory('postgresql', createPostgreeDestroyAdapter, true)
 
+    if (env.USE_GREEN === 'true') {
+      registerGreenUrl(env.GREEN_URL)
+      registerGreenDecoder('snappy', uncompress)
+    }
+
     registerStringLoaders()
     registerServerPlugins()
     this.accountsUrl = env.ACCOUNTS_URL ?? 'http://127.0.0.1:3000'
 
     this.ctx.setWebSocketAutoResponse(new WebSocketRequestResponsePair(pingConst, pongConst))
 
-    this.measureCtx = this.measureCtx = initStatisticsContext('cloud-transactor', {
-      statsUrl: this.env.STATS_URL ?? 'http://127.0.0.1:4900',
-      serviceName: () => 'cloud-transactor: ' + this.workspace,
-      factory: () => new MeasureMetricsContext('transactor', {}, {}, newMetrics(), new CloudFlareLogger())
-    })
+    this.measureCtx = new NoMetricsContext(new CloudFlareLogger())
+    // initStatisticsContext('ctr-' + ctx.id.toString(), {
+    //   statsUrl: this.env.STATS_URL ?? 'http://127.0.0.1:4900',
+    //   serviceName: () => 'cloud-transactor: ' + this.workspace,
+    //   factory: () => new MeasureMetricsContext('transactor', {}, {}, newMetrics(), new CloudFlareLogger())
+    // })
```

Comment:
Do we need to uncomment/remove this?

Reply:
Let it be for now, I supposed this part is not working and causing delays, but I’ve not yet fully sure, will remove on next PR or revert.